frappe/public/js/frappe/views/communication.js

```diff
@@ -677,10 +677,12 @@ frappe.views.CommunicationComposer = Class.extend({
 				last_email_content = last_email_content.slice(0, 20 * 1024);
 			}
 
+			const fwd = this.forward ? "<br><b>Forwarded Message<b><br>" : "";
 			let communication_date = last_email.communication_date || last_email.creation;
 			content = `
 				<div><br></div>
 				${reply}
+				${fwd}
```

Comment:
`reply` contains the signature too, so this will be rendered below the signature.